lib/rubygems/command.rb

```diff
@@ -630,7 +630,7 @@ def wrap(text, width) # :doc:
   Usage:
     gem -h/--help
     gem -v/--version
-    gem command [arguments...] [options...]
+    gem [-C PATH] command [arguments...] [options...]
```

Comment:
Is there a better way to document this type of option?

Reply:
I guess we could add a "Global options:" section below "Usage:"? What do you think?

Reply:
Good idea.